components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -0,0 +1,104 @@
+package batchcheckservice
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger"
+	"github.com/chef/automate/lib/stringutils"
+)
+
+type IBatchCheckService interface {
+	BatchCheck(checks []string, config models.Config) models.BatchCheckResponse
+}
+
+type BatchCheckService struct {
+	CheckTrigger trigger.CheckTrigger
+}
+
+func NewBatchCheckService(trigger trigger.CheckTrigger) IBatchCheckService {
+	return &BatchCheckService{
+		CheckTrigger: trigger,
+	}
+}
+
+func (ss *BatchCheckService) BatchCheck(checks []string, config models.Config) models.BatchCheckResponse {
+	resultMap := make(map[string][]models.CheckTriggerResponse)
+	if shouldRunChecksOnBastion(checks) {
```

Comment:
`shouldRunChecksOnBastion` won't filter any checks, you need to divide the input checks into bastion checks and remote checks.

Reply:
Resolved